tripal/includes/tripal.importer.inc

```diff
@@ -111,11 +111,13 @@ function tripal_get_importer_form($form, &$form_state, $class) {
   // to change the order of their elements in reference to the default ones.
   $form = array_merge($form, $element_form);
 
-  $form['button'] = [
-    '#type' => 'submit',
-    '#value' => t($class::$button_text),
-    '#weight' => 10,
-  ];
+  if (array_key_exists('use_button', $class::$methods) and $class::$methods['use_button'] == TRUE) {
```

Comment:
I'm confused by this line of code.  Shouldn't it be like this:
```
   if (property_exists($class, 'use_button') and $class::$use_button == TRUE) {
```

Reply:
The $class::$methods is an array of methods for providing file uploads/URLs/paths.  I guess the name `methods` is a bit confusing.

Reply:
Ooops! You're right. I won't need property exists through because it's in the base class so it must be there, right?